src/commands/run-file.ts

```diff
@@ -0,0 +1,65 @@
+import * as vscode from "vscode";
+import * as path from "path";
+
+export async function runFile(): Promise<void> {
+  const activeTextEditor: vscode.TextEditor | undefined =
+    vscode.window.activeTextEditor;
+
+  if (!activeTextEditor) {
+    vscode.window.showWarningMessage("Please open a chk file first.");
```

Comment:
```suggestion
    vscode.window.showWarningMessage("Please open a .chk file first.");
```

Does it support markdown? then:

```suggestion
    vscode.window.showWarningMessage("Please open a `.chk` file first.");
```

Reply:
would it be possible to enable this key or dropdown item from command-pallet only when a .chk file is open in editor section? .. just asking

Reply:
Yes possible. Added in https://github.com/chkware/vscode-ext/pull/42/commits/39b3156ee2e1699e22f28e3852596ae5653f8fa1.